src/Microsoft.DotNet.TestFramework/Microsoft.DotNet.TestFramework.TestInstance.cs

```diff
@@ -105,8 +107,13 @@ public TestInstance WithBuildArtifacts()
                                  .Where(file =>
                                  {
                                      file = file.ToLower();
-                                     return file.Contains($"{Path.DirectorySeparatorChar}bin{Path.DirectorySeparatorChar}") 
+
+                                     var isArtifact = file.Contains($"{Path.DirectorySeparatorChar}bin{Path.DirectorySeparatorChar}") 
                                             || file.Contains($"{Path.DirectorySeparatorChar}obj{Path.DirectorySeparatorChar}");
+
+                                     var isBlackListed = BuildArtifactBlackList.Any(b => file.Contains(b.ToLower()));
```

Comment:
True. However, looking at the rest of that class, there's toLower everywhere. I'm going to leave it like this and open an issue about it. Maybe there's other parts of the test framework that do that too. On one hand, it gives you cheap normalization, on the other hand, it may hide bugs.